apps/docs/app/lib/getComponentProps.ts

```diff
@@ -42,6 +42,7 @@ export async function getComponentProps(component: string) {
     const [item] = data;
```

Comment:
![image](https://github.com/user-attachments/assets/36234313-de4e-46d9-8a3c-db098880946d)
This could be fixed in another PR but Listbox should have a background color

Reply:
I can tackle this since it's a listbox issue